test/postControllerTest.js

```diff
@@ -0,0 +1,160 @@
+const express = require('express');
+const request = require('supertest');
+const postController = require('../controllers/postController');
+const sinon = require('sinon');
+const postService = require('../services/postService');
+const assert = require('assert');
+
+const app = express();
+
+app.set('view engine', 'ejs');
+app.use(express.static('../public'));
+
+app.use('', postController);
+
+describe('postController', () => {
+  beforeEach(() => {
+    sinon.stub(postService, 'getAllPosts').resolves([
+      { title: 'post1', author: 'author1', body: 'body1' },
```

Comment:
nit: Your `getAllPosts` doesn't simply resolve an array of objects, but rather an array of instances of `Post`

Reply:
I'm not sure what is it you want me to change here from your comment.

Reply:
It's more correct to change the object `{ title: 'post1', author: 'author1', body: 'body1' }` into an instance of `Post`: `new Post(...)`, but it's not a big deal